src/components/CDPDisplay/index.js

```diff
@@ -50,8 +68,10 @@ function CDPView({ cdpId }) {
   ]);
 
   useEffect(() => {
-    trackCdpById(maker, cdpId, dispatch);
-  }, [cdpId, dispatch, maker]);
+    trackCdpById(maker, cdpId, dispatch).then(() => {
+      if (!cdp.inited) account ? redirect(account) : setCdpAvailable(false);
+    });
```

Comment:
Catching the error here wasn't working because the error was actually being thrown in the feeds reducer. That's now prevented, and we can use the `inited` property to determine if the CDP is unavailable.